snorkel/classification/training/trainer.py

```diff
@@ -240,6 +237,9 @@ def fit(
                 # Update the parameters
                 self.optimizer.step()
 
+                # Update lr using lr scheduler
+                self._update_lr_scheduler(total_batch_num)
```

Comment:
Why is this code block being moved? If it doesn't have to do with an issue introduced by the pytorch upgrade, then would strongly recommend it occur in separate PR.

Reply:
yep, it's related to the upgrade. otherwise, pytorch throws this warning:
```
UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`
```